shared/report.py

```diff
@@ -1,19 +1,34 @@
-from shared.common import *
+from typing import List
+
+from shared.common import Resource, ResourceEdge, message_handler
 from shared.error_handler import exception
 
 
 class Report(object):
+    @exception
+    def general_report(
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Method could be a function](https://app.codacy.com/gh/Cloud-Architects/cloud-discovery/pullRequest?prid=5655786)